HACKING.md

```diff
@@ -101,9 +101,12 @@ can be built using snapcraft either in a LXD container or a multipass VM (or
 natively with `--destructive-mode` on a Ubuntu 16.04 host).
 
 Note: Currently, snapcraft's default track of 5.x does not support building the 
-snapd snap, since the snapd snap uses `build-base: core`, which uses Ubuntu 
-16.04 as the base for building and Ubuntu 16.04 is in Extended Security 
-Maintenance (ESM), and as such only is buildable using snapcraft's 4.x channel.
+snapd snap, since the snapd snap uses `build-base: core`. Building with a 
+`build-base` of core uses Ubuntu 16.04 as the base operating system (and thus 
+root filesystem) for building and Ubuntu 16.04 is now in Extended Security 
```

Comment:
This sentence sounds a bit weird to me but, since Graham is going to edit it later, I didn't think it made sense to nitpick it now